### PR TITLE
Modify the translated PII files in nls folder 20210610

### PR DESCRIPTION
**Pull request by bjcdong**
Files update for JAV21 TVT 1.

Signed-off-by: Dong Chen <bjcdong@cn.ibm.com>

### CODE REVIEW

File: runtime/nls/dump/j9dmp_cs.nls [hunk 1]
```diff
@@ -27,14 +27,14 @@
 # New messages MUST be added at the end of this file.
 # DO NOT delete messages from this file, as it will change their indices.
 # If you wish to remove a message, delete its text, but leave the key in place
-# Important: the NLS messages are shared between all releases including branched SRs.
+# Important: the NLS messages are shared between all releases including branched SRs.
```
Comment (pshipton) [inline runtime/nls/dump/j9dmp_cs.nls:30]: Please remove all the extra spaces. There are a number of them added in this file.

File: runtime/nls/j9vm/j9vm_ca.nls [hunk 2]
```diff
@@ -1704,10 +1704,6 @@ J9NLS_VM_ILLEGAL_ACCESS_NONSTATIC_FINAL.user_response=Modify source code to avoi
 # Fourth argument is nest host name
 J9NLS_VM_NEST_HOST_HAS_DIFFERENT_CLASSLOADER=La classe %2$.*1$s i l'amfitri\u00f3 d'imbricaci\u00f3 %4$.*3$s han de tenir el mateix carregador de classes.
 # START NON-TRANSLATABLE
-J9NLS_VM_NEST_HOST_HAS_DIFFERENT_CLASSLOADER.sample_input_1=3
```
Comment (pshipton) [inline runtime/nls/j9vm/j9vm_ca.nls:1707]: These lines should not be removed. Here and below, and in the subsequent files as well.

File: runtime/nls/j9vm/j9vm_ca.nls [hunk 3]
```diff
@@ -1924,7 +1904,7 @@ J9NLS_VM_CLASS_LOADING_ERROR_EXTEND_FINAL_SUPERCLASS.user_response=Contact the p
 # arguments 1 and 2 are the specified interface name
 J9NLS_VM_CLASS_LOADING_ERROR_SUPERCLASS_IS_INTERFACE=Superclasse %2$.*1$s no pot ser una interf\u00edcie
 # START NON-TRANSLATABLE
-J9NLS_VM_CLASS_LOADING_ERROR_SUPERCLASS_IS_INTERFACE.sample_input_1=13
+J9NLS_VM_CLASS_LOADING_ERROR_SUPERCLASS_IS_INTERFACE.sample_input_1=9
```
Comment (pshipton) [inline runtime/nls/j9vm/j9vm_ca.nls:1907]: The original value is correct, pls restore it. Here and below, and in the subsequent files.

File: runtime/nls/jitm/j9jit_it.nls [hunk 4]
```diff
@@ -160,6 +160,14 @@ J9NLS_RELOCATABLE_CODE_METHOD_TRAMPOLINE_MISMATCH.user_response=Destroy and recr
 J9NLS_RELOCATABLE_CODE_METHOD_TRAMPOLINE_MISMATCH.link=dita:///diag/appendixes/cmdline/cmdline_x.dita#cmdline_x/xshareclasses-reset
 # END NON-TRANSLATABLE
 
+J9NLS_RELOCATABLE_CODE_FSD_MISMATCH=Mancata corrispondenza funzione FSD abilitata. Il codice AOT nella cache di classi condivisa verr\u00e0 ignorato.
```
Comment (pshipton) [inline runtime/nls/jitm/j9jit_it.nls:163]: This message should not be added back here, it's properly added at the end now. Here and in the subsequent files.

File: runtime/nls/jitm/j9jit_it.nls [hunk 5]
```diff
@@ -355,32 +363,24 @@ J9NLS_JIT_OPTIONS_LARGE_PAGE_SIZE_NOT_SUPPORTED_WITH_PAGETYPE.sample_input_6=pag
 
 J9NLS_RELOCATABLE_CODE_HEAP_BASE_FOR_BARRIER_RANGE_MISMATCH=Mancata corrispondenza funzione Heap Base For Barrier Range.Il codice AOT nella cache di classi condivisa verr\u00e0 ignorato.
 # START NON-TRANSLATABLE
-J9NLS_RELOCATABLE_CODE_HEAP_BASE_FOR_BARRIER_RANGE_MISMATCH.explanation=The JVM is not running in a compatible heap configuration as the one that generated the code in the SCC.
-J9NLS_RELOCATABLE_CODE_HEAP_BASE_FOR_BARRIER_RANGE_MISMATCH.system_action=The compiler will not generate AOT code. AOT code from the shared class cache will not be used.
-J9NLS_RELOCATABLE_CODE_HEAP_BASE_FOR_BARRIER_RANGE_MISMATCH.user_response=Destroy and recreate the AOT code in the shared class cache for the current platform using the -Xshareclasses:reset option.
-J9NLS_RELOCATABLE_CODE_HEAP_BASE_FOR_BARRIER_RANGE_MISMATCH.link=dita:///diag/appendixes/cmdline/cmdline_x.dita#cmdline_x/xshareclasses-reset
+J9NLS_RELOCATABLE_CODE_TM_MISMATCH.explanation=The JVM is not running in a compatible heap configuration as the one that generated the code in the SCC.
```
Comment (pshipton) [inline runtime/nls/jitm/j9jit_it.nls:366]: The original names are correct, pls restore them. Here and below, and in the subsequent files.

**Comment by pshipton** [comment]
This is replaced by https://github.com/eclipse-openj9/openj9/pull/12933

**Comment by bjcdong** [comment]
Close this PR as this is replaced by #12933.

**Pull request closed without merging**